src/script/components/MessagesList/Message/FailedToAddUsersMessage.test.tsx

```diff
@@ -32,135 +32,160 @@ import {FailedToAddUsersMessage} from './FailedToAddUsersMessage';
 
 setStrings({en});
 
-const createFailedToAddUsersMessage = (partialFailedToAddUsersMessage: Partial<FailedToAddUsersMessageEntity>) => {
-  const failedToAddUsersMessage: Partial<FailedToAddUsersMessageEntity> = {
-    displayTimestampLong: () => '',
-    displayTimestampShort: () => '',
-    timestamp: ko.observable(Date.now()),
-    unsafeSenderName: ko.pureComputed(() => ''),
-    ...partialFailedToAddUsersMessage,
-  };
-  return failedToAddUsersMessage as FailedToAddUsersMessageEntity;
+const createFailedToAddUsersMessage = ({
+  users,
+  reason = AddUsersFailureReasons.UNREACHABLE_BACKENDS,
+  backends = [],
+}: {
+  users: QualifiedId[];
+  reason?: AddUsersFailureReasons;
+  backends?: string[];
+}) => {
+  return new FailedToAddUsersMessageEntity(users, reason, backends, Date.now());
 };
 
+function createUser(qualifiedId: QualifiedId, name: string) {
+  const user = new User(qualifiedId.id, qualifiedId.domain);
+  user.name(name);
+  return user;
+}
+
 describe('FailedToAddUsersMessage', () => {
   it('shows that 1 user could not be added', async () => {
     const userState = new UserState();
     const [qualifiedId1] = generateQualifiedIds(1, 'test.domain');
 
-    const user1 = new User(qualifiedId1.id, qualifiedId1.domain);
+    const user1 = createUser(qualifiedId1, 'Felix');
     userState.users.push(user1);
 
     const message = createFailedToAddUsersMessage({
-      qualifiedIds: [qualifiedId1],
+      users: [qualifiedId1],
     });
 
-    const {getByTestId} = render(
+    const {getAllByText} = render(
       withTheme(<FailedToAddUsersMessage isMessageFocused message={message} userState={userState} />),
     );
 
-    const elementMessageFailedToAdd = getByTestId('element-message-failed-to-add-users');
-    expect(elementMessageFailedToAdd.getAttribute('data-uie-value')).toEqual('1-user-not-added');
+    const mainMessage = getAllByText(
+      (_, element) =>
+        element?.textContent ===
+        'Felix could not be added to the group as the backend of test.domain could not be reached.',
+    );
+    expect(mainMessage.length).toBeGreaterThanOrEqual(1);
```

Comment:
I did change the tests to actually test what the end user is supposed to see on screen. 
Using `testId` is usually a weak way of testing (as it's not really testing end user behaviors)